### ISSUE TITLE
Using preview packages result in negative libyears

### ISSUE THREAD

**Issue by henrikrxn** [ecoAPM/dotnet-libyear#77]
**libyear version used**
```
> dotnet tool list --global
libyear                   6.0.0        dotnet-libyear
```

**Steps to reproduce**
1. Repro for a .NET Framework project is available here https://github.com/henrikrxn/PreviewNegativeLibyear
2. The only NuGet package is a preview, which is the latest on nuget.org
3. Clone repro example
4. `dotnet libyear` in the root of the clone

**Expected behavior**
- Age of the the specific package to be 0.0 
- Project is 0,0 libyears behind

**Observed behavior**
- Age is -2.1
- Project is -2,1 libyears behind

**Comment by SteveDesmond-ca** [comment]
Thanks for this bug report, I'll take a look and hopefully get a patch out in the next week or so.

**Comment by SteveDesmond-ca** [comment]
Thanks again, @henrikrxn -- this has been fixed in v6.0.1 (via 6aae1dc), which is being published to NuGet this very instant!
